src/main/scala/overflowdb/codegen/CodeGen.scala

```diff
@@ -725,6 +734,14 @@ class CodeGen(schemaFile: String, basePackage: String) {
                    |private var _${containedNode.localName}: List[$containedNodeType] = Nil
                    |def ${containedNode.localName}: List[$containedNodeType] = this._${containedNode.localName}
                    |""".stripMargin
+              case Cardinality.ISeq =>
+                s"""
+                   |private var _${containedNode.localName}: Array[$containedNodeType] = null
+                   |def ${containedNode.localName}: scala.collection.immutable.IndexedSeq[$containedNodeType] =
```

Comment:
Please stop using fully qualified names. It is just ugly and makes reading harder. If you are just writing `immutable.IndexedSeq` and put and import up top everybody knows what you are talking about.